ReactNativeClient/lib/permissions.ts

```diff
@@ -0,0 +1,9 @@
+const { PermissionsAndroid } = require('react-native');
```

Comment:
Please name the file "checkPermissions.ts" as this is how you import it below. In general the filename should the same as the exported item, to make searching in the code base easier. I'll see if there's an eslint rule for this to make it clearer.